specifications/api/swagger.yaml

```diff
@@ -589,6 +578,46 @@ x-stakePoolDesirability: &stakePoolDesirability
     The most desirable pool gets rank 1, the second most desirable pool gets rank 2 and so on.
   example: 42
 
+x-non-myopic-member-rewards: &nonMyopicMemberRewards
+  <<: *amount
+  description: |
+    Non-myopic member rewards is the recomended way to rank stake pools. It is the rewards the
```

Comment:
Perhaps this is a weird way to describe it.

E.g. for what unit of time do the rewards correspond to? (I think it is an epoch.)

Reply:
Non-myopic member rewards = rewards one can expect at the end of an epoch from a fully saturated pool.

Reply:
I rephrased in 44e6ff870c7c0805661b425d9ca934b9892a576b 

> epoch from a fully saturated pool

That is misleading though? It is assuming the top k pools are saturated, and the rest are empty (except the user's stake… and pledge?), or something like that. I didn't mention on those specifics in the commit.